README.md

```
@@ -1,106 +1,67 @@
+  </a>
+</p>
+
+# TrogonDB
 
-EventStoreDB is the event-native database, where business events are immutably stored and streamed. Designed for event-sourced, event-driven, and microservices architectures
+TrogonDB is the event-native database, where business events are immutably stored and streamed. Designed for event-sourced, event-driven, and microservices architectures
 
-- [What is EventStoreDB ](#what-is-eventstoredb)
-- [What is EventStore Cloud ](#what-is-event-store-cloud)
-- [Licensing](#licensing)
+- [What is TrogonDB ](#what-is-TrogonDB)
```

markdownlint-cli2 (0.17.2)

Spaces inside link text
null

(MD039, no-space-in-links)